src/Core/Scheduler/Providers/WMTS_Provider.js

```diff
@@ -85,18 +85,19 @@ WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer, ta
     const url = this.url(coordWMTS, layer);
 
     return OGCWebServiceHelper.getXBilTextureByUrl(url, layer.networkOptions).then((texture) => {
-        const { min, max } = OGCWebServiceHelper.ioDXBIL.computeMinMaxElevation(
-            texture.image.data,
-            SIZE_TEXTURE_TILE, SIZE_TEXTURE_TILE,
-            pitch);
+        const { min, max } = (texture.min == undefined && texture.max == undefined) ? texture :
+            OGCWebServiceHelper.ioDXBIL.computeMinMaxElevation(
```

Comment:
if `getXBilTextureByUrl` call `io_driverxbil.read` that calls `parseXbil` that calls `computeMinMaxElevation`, what's the point of this call? It is guaranteed we have already done it. We could simply remove it.

Reply:
yes